boot/bootutil/src/image_validate.c

```diff
@@ -280,6 +287,8 @@ bootutil_find_key(uint8_t *keyhash, uint8_t keyhash_len)
     const struct bootutil_key *key;
     uint8_t hash[IMAGE_HASH_SIZE];
 
+    BOOT_LOG_DBG("bootutil_find_key:");
```

Comment:
likewise here, the : makes it look like there is some expected output but other ones above do not have it when there is nothing to output